analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/PermissiveCorsTest.cs

```diff
@@ -76,7 +76,7 @@ public class PermissiveCorsTest
                 .Verify();
 
         private static IEnumerable<MetadataReference> AdditionalReferences =>
-            NuGetMetadataReference.MicrosoftNetHttpHeaders(Constants.NuGetLatestVersion)
+            NuGetMetadataReference.MicrosoftNetHttpHeaders("2.1.14")
```

Comment:
will you open a ticket to bring this back to NuGet Latest?

why: we keep LatestVersion to be aware if some API changes lead to our rules creating FPs or FNs  (e.g. a deprecated attribute might be removed, so we won't raise anymore on net8 and we need to update the rule)